blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubOrganizationFolder.java

```diff
@@ -32,10 +34,12 @@ public boolean isScanAllRepos() {
             SCMNavigator scmNavigator = getFolder().getSCMNavigators().get(0);
             if(scmNavigator instanceof GitHubSCMNavigator){
                 GitHubSCMNavigator gitHubSCMNavigator = (GitHubSCMNavigator) scmNavigator;
-                return (StringUtils.isBlank(gitHubSCMNavigator.getIncludes()) || gitHubSCMNavigator.getIncludes().equals("*"))
-                        && StringUtils.isBlank(gitHubSCMNavigator.getExcludes())
-                        && (StringUtils.isBlank(gitHubSCMNavigator.getPattern())
-                        || gitHubSCMNavigator.getPattern().equals(".*"));
+                for (SCMTrait<?> t: gitHubSCMNavigator.getTraits()) {
+                    if (t.getDescriptor().getClass().getAnnotation(Selection.class) != null) {
+                        // approximate logic
```

Comment:
I say approximate as *technically* the user could add a useless `new WildcardSCMSourceFilterTrait("*","")` which would not be doing any filtering but we would be returning `false`